### DISCUSSION TITLE
How can I synch my notebook from Dropbox

### DISCUSSION THREAD

**Discussion by mattcristal** [orgzly-revived/orgzly-android-revived#116] [Q&A] [answered]
Good morning,

I am an Orgzly user and wanted to make the switch to Orgzly revived. Great to hear that you are working on it!

However, I don't understand how can I synch my notebook from Dropbox, as I do in Orgzly.

I am attaching some screenshots here, would really appreciate some help!

This is how I do it in Orgzly:
![orgzly](https://github.com/orgzly-revived/orgzly-android-revived/assets/33809495/07795ca4-3666-4da9-81a5-860f7b912419)

And this is what I see in Orgzly revived:
![orgzly_revived](https://github.com/orgzly-revived/orgzly-android-revived/assets/33809495/71fc0bca-0de2-4464-996c-ed22bbdd5c6f)

**Comment by amberin** [comment, marked as answer]
@mattcristal Hello and welcome!

If you use the Fdroid build, Dropbox repos have always been disabled, I believe for code license reasons.

You need to install the "premium" build which is intended for (but not yet on) Play store. You can find the APK among the [release assets in this repo](https://github.com/orgzly-revived/orgzly-android-revived/releases/tag/v1.8.14). Or you can install via the awesome [Obtainium](https://github.com/ImranR98/Obtainium) app, which will let you choose between the two builds.

**Reply by mattcristal** [reply]
Thanks!

In the end, I decided to opt for F-DROID so that I can upgrade the version easily and use a local notebook that I will backup via another cloud service.

Thanks again!

**Reply by pihentagy** [reply]
Oh, somewhere it should be stated. I struggled also to set up Dropbox sync.

What are my options if I don't want to install full Dropbox client, but would like to have dropbox sync? Okay, a git(hub) sync would also be a great thing to have instead.

**Reply by amberin** [reply]
For native Dropbox support in Orgzly, I know three options:
- install via Google Play
- install via Obtainium and don't choose the "fdroid" build
- install from Github releases and don't choose the "fdroid" build.

**Reply by pihentagy** [reply]
Is Orgzly revived available from Google Play?

I found now fdroid build "weird".

**Reply by amberin** [reply]
> Is Orgzly revived available from Google Play?
> 
> I found now fdroid build "weird".

@pihentagy Yes, please see instructions here: https://github.com/orgzly-revived/orgzly-android-revived/discussions/250

**Comment by torfjelde** [comment]
Just a heads up: I just tried using the latest release with Dropbox and clicking the `Link to Dropbox` button simply brings me back to the `Repositories` page and does nothing :confused:
 
 I don't have time to do any digging right now myself (sorry!), but figured I'd just post this quick comment in case it's something obvious going or to follow up later:)
 
 (Btw, really appreciate this effort to keep Orgzly going!)

**Reply by amberin** [reply]
Thanks for the info. I hope someone else who uses Dropbox can confirm if it's broken for more people or not, and/or file an issue.
